### ISSUE TITLE
The question about the loss

### ISSUE THREAD

**Issue by EricKani** [ai-tor/PyTorchSRGAN#13]
Hi,
I just changed you code to fit my own dataset. But I got some problem about my training loss... 

`[4/100][2112/2112] Discriminator_Loss: 0.9368 Generator_Loss (Content/Advers/Total): 0.0021/1.2429/0.0033`
`[5/100][2112/2112] Discriminator_Loss: 0.8507 Generator_Loss (Content/Advers/Total): 0.0022/1.4514/0.0036`
`[6/100][2112/2112] Discriminator_Loss: 0.8301 Generator_Loss (Content/Advers/Total): 0.0022/1.5216/0.0037`

We can see that the Discriminator_Loss is decreasing, but the Generator_Loss got increased...I am confused by this problem and I am a new user about gannet. I hope I can get some help from you.
Thank you very much!!
